src/pages/map.tsx

```diff
@@ -178,6 +178,7 @@ const MapPage = ({ schedule }: Props) => {
               <IconButton aria-label="Back to Home" as="a" borderRadius={4} icon={<ArrowBackIcon />} />
             </Link>
             <Select
+              color="white"
```

Comment:
```suggestion
```

Remove this, change the `bg` on the containing `Box` so that it follows dark/light theme.